### ISSUE TITLE
Support more than 9 tags

### ISSUE THREAD

**Issue by JustinLovinger** [MaxVerevkin/i3bar-river#9]
River supports up to 32 tags, but only tags up to "9" are displayed in `i3bar-river`. As far as I can tell, the culprit is `1..10` here, <https://github.com/MaxVerevkin/i3bar-river/blob/master/src/wm_info_provider/river.rs#L117>, which should be `1..=32`.

**Comment by MaxVerevkin** [comment]
It would be nice to support that. The reason it currently shows only first 9 tags it because the upper tags are used by some as "scratch" tags.

**Comment by JustinLovinger** [comment]
The number of tags to show could be an option, for users who want to hide a subset of tags.

**Comment by MaxVerevkin** [comment]
Yeah. Something like

```toml
[wm.river]
max_tag = 9 # the default
max_tag = 32 # show all tags
```

**Comment by MaxVerevkin** [comment]
Implemented in 432b6d6
